public/editor/scripts/ui/publisher.js

```diff
@@ -33,10 +33,11 @@ function Publisher() {
       update: $("#publish-button-update"),
       unpublish: $("#publish-button-unpublish"),
       parent: $("#publish-buttons"),
-      indexMessage: $("#no-index")
+      indexMessage: $("#no-index"),
```

Comment:
You have a stray comma here.  The last element in a list like this shouldn't get a dangling comma.